docs/scenarios_authoring/Spel.md

```diff
@@ -95,15 +95,18 @@ There are a few notable exceptions:
 | `{:}`                  | an empty record                | Record{}             |
 | `{john:300, alex:400}` | a map (name-value collection)  | Map[String, Integer] |
 | `#input`               | variable                       |                      |
+| `'AA' + 'BB'`          | "AABB"                         |             String   |
                                     
-## Arithmetic Operators
+### Arithmetic Operators
 
-| Expression    | Result   | Type     |
```

Comment:
Why do you remove plus operator and move it to Basics. This operator is too easy to be explained in this section? :)